listenbrainz/model/external_service_oauth.py

```diff
@@ -0,0 +1,60 @@
+from listenbrainz.model import db
+from listenbrainz.webserver.admin import AdminModelView
+
+
+class ExternalService(db.Model):
```

Comment:
we should have a further discussion about this - if we have to make a sqlalchemy model for the admin anyway, then it seems a bit silly to have tens or hundreds of lines of code with hand-written SQL for really standard "update this row to change a value" when we can do the same thing in 2 lines of ORM code without reducing readability (imo). Not a task for this PR, but let's open a ticket to talk more about it.

Reply:
Completely agree. There are many places where we could benefit using the ORM.